native/src/components/Header.tsx

```diff
@@ -179,7 +179,11 @@ const Header = ({
 
   const visible = showItems && !!goToLanguageChange
   const items = [
-    renderItem(HeaderButtonTitle.Search, 'search', visible, () => navigation.navigate(SEARCH_ROUTE)),
+    renderItem(HeaderButtonTitle.Search, 'search', visible, () =>
+      navigation.navigate(SEARCH_ROUTE, {
+        searchText: '',
```

Comment:
I think it is cleaner and less error prone, to set the default in SearchModalContainer, as it is allowed the searchText is null or undefined the component should handle these cases.
`          initialSearchText={route.params.searchText ?? ''}`